### PR TITLE
Fixes #26401 - Remove GPG key from f-release-client

### PR DESCRIPTION
**Pull request by ekohl**
This file belongs to foreman-release and can create conflicts on upgrade.

### CODE REVIEW

File: packages/foreman/foreman-release/foreman-client.repo [hunk 1]
```diff
@@ -3,11 +3,11 @@ name=Foreman client nightly
 baseurl=https://yum.theforeman.org/client/nightly/$DIST/$basearch
 enabled=1
 gpgcheck=0
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman-client
```
Comment (evgeni) [inline packages/foreman/foreman-release/foreman-client.repo]: where is this file coming from? I don't see it in `%files`

Comment (ekohl) [reply packages/foreman/foreman-release/foreman-client.repo]: The repo is currently not signed (hence the `gpgcheck=0`). I just thought I should rename this to make it clear it's a different file. Users have been setting gpgcheck=1.

Comment (evgeni) [reply packages/foreman/foreman-release/foreman-client.repo]: oooh, I'll be damned for not seeing the `gpgcheck=0` above.

Comment (ekohl) [reply packages/foreman/foreman-release/foreman-client.repo]: See https://projects.theforeman.org/issues/26278 as well. I also consider leaving the line out.

**Pull request merged**